opentelemetry-distro/pyproject.toml

```
@@ -25,7 +25,7 @@ classifiers = [
 dependencies = [
   "opentelemetry-api ~= 1.12",
   "opentelemetry-instrumentation == 0.34b0",
-  "opentelemetry-sdk == 1.13.0",
+  "opentelemetry-sdk ~= 1.13.0",
```

This is equivalent to: opentelemetry-sdk >= 1.13.0, == 1.13.* but what ideally should be ~= 1.13
```suggestion
  "opentelemetry-sdk ~= 1.13",
```

---

good call out! :-)